subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/java/JavaLibraryPlugin.java

```diff
@@ -46,16 +50,37 @@ public void execute(CopySpec copy) {
             }
         });
 
+
+        final File dependenciesFile = new File(project.getBuildDir(), "shipkit-dependencies");
+
+        Task copyDepsToFile = TaskMaker.task(project, "createDependenciesFile", CreateDependenciesFile.class, new Action<CreateDependenciesFile>() {
+            @Override
+            public void execute(CreateDependenciesFile task) {
+                task.setDescription("Creates file with resolved runtime dependencies.");
+                task.setOutputFile(dependenciesFile);
+                task.setConfiguration(project.getConfigurations().getByName("default"));
+            }
+        });
+
+        final CopySpec dependencyFile = project.copySpec(new Action<CopySpec>() {
```

Comment:
CopySpec feels like an overkill here, why not just adding "from" to the "sourcesJar" task?